src/main.cpp

```diff
@@ -2145,6 +2151,10 @@ void loop() {
         loopLED();
     }
 
+    if (WATER_SENS) {
+        loopWATER();
```

Comment:
This should definitely not be called on every loop. This is just a waste of resources.

Reply:
Called now every 100ms